imsim/stamp.py

```diff
@@ -84,6 +84,27 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         Returns:
             xsize, ysize, image_pos, world_pos
         """
+
+        # First do a parsing check to make sure that all the options passed to
+        # the config are valid, and no required options are missing.
+
+        req = {'fft_sb_thresh': float,
+               'band': str,
+               'world_pos': galsim.CelestialCoord,
```

Comment:
Not required.  (And probably not worth mentioning in opt either.  This is usually automatically computed by GalSim.)

Reply:
So, I added that because I was getting:

```
galsim.errors.GalSimConfigError: Unexpected attribute world_pos found
```

as it is in our imsim-config.yaml.  

I can add it to the opt list.

Reply:
Or perhaps it is already in a global ignore list and the option to the call would fix it?

Reply:
Ah... yes, that fixed it. Thanks.